pixeltable/exprs/data_row.py

```diff
@@ -281,6 +281,18 @@ def flush_img(self, index: int, col: Optional[catalog.Column] = None) -> None:
             pass
         self.vals[index] = None
 
+    def move_tmp_media_file(self, index: int, col: catalog.Column) -> None:
+        """Move a media file from a temporary location to the MediaStore"""
```

Comment:
This isn't quite what it does and the comment may be confusing. What it really does is *conditionally* move a media file *if* it's in temp storage.